crates/holochain_types/CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## \[Unreleased\]
 
+- Added `Default` implementation for `holochain_types::prelude::InstallAppPayload`. You may add `..Default::default` 
+  after other fields if you use this type so that the type can be updated without breaking your code.
```

Comment:
```suggestion
  after other fields if you use this type so that the type can be updated without breaking your code. [#2745](https://github.com/holochain/holochain/pull/2745)
```